README.md

```diff
@@ -31,20 +32,60 @@ All inputs are optional. But depending of the authentication type chosen, follow
 
 Since this action requires sensitive information such as user name, password and encoded certificate for example, it would be ideal to store them securely. We can achieve this in a GitHub repo by using [secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets). So, click on `settings` tab in your repo and add:
 
+- 2 new secrets for the Entra ID identity:
+  - `APP_ID` - store App ID of the Entra application to use for authentication
+  - `TENANT` - store the ID of the tenant where the Entra application is located
+
 - 2 new secrets if `authType` is `password`:
   - `ADMIN_USERNAME` - store the admin user name in this (e.g. user@contoso.onmicrosoft.com)
   - `ADMIN_PASSWORD` - store the password of that user in this.
 
 - 2 new secrets if `authType` is `certificate`:
-  - `CERTIFICATE_ENCODED` - store the Base64-encoded string of the certificate stored in the Azure AD application
+  - `CERTIFICATE_ENCODED` - store the Base64-encoded string of the certificate stored in the Entra application
   - `CERTIFICATE_PASSWORD` - store the certificate password
 
-- 2 new secrets if using a custom Azure AD identity:
-  - `APP_ID` - store App ID of the Azure AD application to use for authentication
-  - `TENANT` - store the ID of the tenant from which accounts should be able to authenticate
-
 These secrets are encrypted and can only be used by GitHub actions.
 
+### Example workflow - CLI for Microsoft 365 Login (federated identity authentication)
+
+On every `push` build the code and then log in to Microsoft 365 before deploying, using federated identity authentication.
+
+```yaml
+name: SPFx CICD with Cli for Microsoft 365
+
+on: [push]
+
+jobs:
+  build:
+    ##
+    ## Build code omitted
+    ##
+
+  deploy:
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x]
```

Comment:
Since the recommended version of CLI for Microsoft 365 is Node 22, shouldn't we use this one instead?